src/components/ConfigSlider/index.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react';
+import Slider from '@react-native-community/slider';
+
+import {
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+} from '../Globals/ConfigMetric';
+
+import {
+  Button,
+  Modal,
+  ModalContent,
+  ModalWrapper,
+  ButtonText,
+  ModalButton,
+  ModalButtonWrapper,
+  PercentageDisplay,
+  ModalOpenButton
+} from './styles';
+import { View } from 'react-native';
+
+interface ConfigSlider {
+  label: string
+}
+
+const ConfigSlider = ({ label } : ConfigSlider) => {
```

Comment:
Is this component still needed since we are not using anymore? I think we can delete uses of it as well as the `slider` package that came with it